client/pages/car-details.jsx

```diff
@@ -100,28 +113,42 @@ class CarDetails extends React.Component {
     }
     return (
       <>
-        <div className="row my-3 rounded overflow-hidden">
-            <div className="col-lg-11">
+        <div className="row mt-3 rounded overflow-hidden">
+          <div className="col-lg-12">
             <h1 className='py-3 work-sans fw-bold text-capitalize'>{year} {make} {model}</h1>
-            <Card.Img className='shadow p-0 mb-3' src={photoUrl} alt="" />
-            <div className='m-0 overflow-hidden rounded'>
-              <div className="row py-2 mx-0 bg-navbar-menu">
-                <h2 className='col text-start'>Recent Records</h2>
-                <div className="col text-end">
-                  <a href="" onClick={this.toggleModal} className='text-reset'><i className="fs-3 bi bi-plus-circle pe-2"></i></a>
-                </div>
+            <div className="row">
+              <div className="col-lg-9">
+                <Card.Img className='shadow p-0 mb-3 rounded' src={photoUrl} alt="" />
+              </div>
+              <div className='col-lg-3 mb-3 ps-lg-0'>
+                <Card className='h-100'>
+                  <Card.Header className='bg-navbar-menu work-sans fs-3 py-md-3'>Next Oil Change</Card.Header>
+                  <Card.Body className='row body-sans fs-1 py-sm-5 py-md-5 py-5'>
+                    <p className='m-auto'>
+                      {this.getNextOilChange() ? this.getNextOilChange().toLocaleString() + ' Miles' : 'No Past Oil Changes'}
+                    </p>
```

Comment:
Instead of calculating the oil change twice, calculate it once before your `return` statement and store the return value in a variable.